UI/MapboxNavigationUI/Placemark.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import MapboxGeocoder
+import Mapbox
+
+let distanceFormatter = DistanceFormatter()
+
+extension MapboxGeocoder.GeocodedPlacemark {
```

Comment:
Is there a conflict that would force us to qualify `GeocodedPlacemark` with the `MapboxGeocoder` module here?